jlm/tooling/CommandLine.cpp

```diff
@@ -919,6 +935,14 @@ JlmOptCommandLineParser::ParseCommandLineArguments(int argc, char ** argv)
               aASteensgaardRegionAware,
               JlmOptCommandLineOptions::ToCommandLineArgument(aASteensgaardRegionAware),
               "Steensgaard alias analysis with region-aware memory state encoding"),
+          ::clEnumValN(
+              aAAndersenAgnostic,
+              JlmOptCommandLineOptions::ToCommandLineArgument(aAAndersenAgnostic),
+              "Andersen alias analysis with agnostic memory state encoding"),
+          ::clEnumValN(
+              aAAndersenRegionAware,
+              JlmOptCommandLineOptions::ToCommandLineArgument(aAAndersenRegionAware),
+              "Andersen alias analysis with region-aware memory state encoding"),
```

Comment:
Could we push this over the Steensgaard flags such that they are sorted alphabetically in the `jlm-opt --help` output?